ios/Trifle/Sources/TrifleCertificate.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  TrifleCertificate.swift
+//  Trifle
+//
+
+import Foundation
+import Wire
+
+public class TrifleCertificate : Codable {
+    private let proto: Certificate
+    
+    public init(data: Data) throws {
+        self.proto = try ProtoDecoder().decode(Certificate.self, from: data)
+    }
+    
+    internal func getCertificate() -> Certificate {
```

Comment:
maybe instead of `getCertificate`, what do you think of `deserialize`

Reply:
we already have a decoder because of codable. deserialize would be too confusing i think. GetCertificate is more accurate i think.

Reply:
TrifleCertificate#getCertificate seems a little overloaded I feel

Reply:
I hear you and am open to other names. I ended up on getCertificate because the return type is Certificate. I tried getCertData, getRawCert, serialize, getValue...
Any other suggestions?